ebmdatalab/maps.py

```diff
@@ -1,131 +1,142 @@
+        :param mapping_data_path (str): the path of the json file
+        :param index_file_path (str): path of the index file. Index file contains additionally information
+                about the geographical areas such as London False/True
+        """
+
+        self.map_data = gpd.read_file(mapping_data_path)
+        if index_file_path is None:
+            final = gpd.GeoDataFrame(
+                pd.merge(left=data, right=self.map_data, how='inner', left_on='row_id', right_on='code'))
+            self.final_df = final
+        else:
+            self.index_file = pd.read_csv(index_file_path)
+            interim = gpd.GeoDataFrame(
+                pd.merge(left=data, right=self.index_file, how='inner', left_on='row_id', right_on='code',
+                         left_index=True))
+            final = gpd.GeoDataFrame(
+                pd.merge(left=interim, right=self.map_data, how='inner', left_on='row_id', right_on='code'))
+            self.final_df = final
+
+    def create_map_with_london_sep(self, plot_value, title, description=None,
+                   size=(12, 10),
+                   cmap="OrRd",
+                   legend=True, fit_legend=False,
+                   edgecolor="black", linewidth=0.1, save=False):
+        """
+        create plot with london separate
+
+        :param plot_value (str): points to which column in the dataframe for plotting
+        :param title (str): title of the plots
+        :param description (str): description of the plots
+        :param london_separate (bool): True or False
+        :param size (2 item tuple): size of the plot
+        :param cmap (str): colors - standard use is OrRd
+        :param legend (bool): True or False
+        :param fit_legend (bool): True or False. Scales the legend to the size of the map
+        :param edgecolor (str): Set to black
+        :param linewidth (float): Width of the line around geographical areas
+        :param save (bool): True or False. If True, saves in self.plots as a dictionary with description as key, plot as value
+        :return: Figure
+        """
+
         subplot_spec = gridspec.GridSpec(1, 1)[0]
-    if separate_london:
-        if london_layout == "horizontal":
-            gs = gridspec.GridSpecFromSubplotSpec(
-                nrows=1, ncols=2, width_ratios=[1, 2], subplot_spec=subplot_spec
-            )
-            ldn_ax = fig.add_subplot(gs[0, 0])
-            roe_ax = fig.add_subplot(gs[0, 1])
+        lon_data = self.final_df[self.final_df["is_london"] == True]
+        roe_data = self.final_df[self.final_df["is_london"] == False]
+        gs = gridspec.GridSpecFromSubplotSpec(nrows=1, ncols=2, width_ratios=[1, 2], subplot_spec=subplot_spec)
+        fig = plt.figure(figsize=size)
+        ldn_ax = fig.add_subplot(gs[0, 0])
+        ldn_ax.axis("off")
+        ldn_ax.set_title("London:\n{}".format(title), size=14)
+        roe_ax = fig.add_subplot(gs[0, 1])
+        if fit_legend == True:
```

Comment:
Again, another super trivial point that there's no need to change now but it's usually considered more "Pythonic" to write `if condition:` rather than `if condition == True:`.

Partly, I suppose, it's because it means exactly the same thing and is slightly shorter. But also because the Python community seems to value code which approximates natural language in some sense (hence using `and` and `not` rather that `&&` and `!`).